### PR TITLE
#574 added GCP NEXT events as new event series.

### LINKED ISSUE #574: Add "GCP" as new event series.
Google is organizing GCPNEXT event and GDG chapters are organizing GCPNEXT Extended event.

In Version 2.4 we should include that into our events section.

### PR DESCRIPTION
**Pull request by ouzman**
Fixes #574 
Page and drawer added like screenshots. Colors choosen via https://hub.gdgx.io/api/v1/tags/gcpnext

img-20160314-wa0003

img-20160314-wa0002

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -44,6 +44,8 @@
   <color name="color_primary_dark_wtm">#3D9B86</color>
   <color name="color_primary_ioextended">#50B8D0</color>
   <color name="color_primary_dark_ioextended">#30A1BC</color>
+  <color name="color_primary_gcpnext">#E74235</color>
```
Comment (tasomaniac) [inline app/src/main/res/values/colors.xml]: Are they the same colors used in firefly project? Those colors actually come from hub Api.

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -44,6 +44,8 @@
...
+  <color name="color_primary_dark_gcpnext">#be352b</color>
```
Comment (tasomaniac) [inline app/src/main/res/values/colors.xml]: Can you make this upper case?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -102,11 +102,13 @@
   <string name="order_by_distance">Sort by distance</string>
   <string name="gcm_event_ticker">A new event at your home GDG has been scheduled.</string>
   <string name="ioextended_description">Google I/O Extended is about joining the Google conference at your local group.</string>
+  <string name="gcpnext_description">GCP NEXT Extended is about joining the Google conference at your local group.</string>
```
Comment (tasomaniac) [inline app/src/main/res/values/strings.xml]: Can you change the description of the event to mention about the conference. Everyone knows Google IO but not much people know gcp

**Comment by tasomaniac** [comment]
Wow! It's good to see familiar faces here :) Thank you for the contribution. 

Can you also add screenshots of the drawer and the page?

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (ouzman) [inline app/src/main/res/values/colors.xml]: No, there are my choices :) I checked colors of WTM and Devfest. They are not same with Hub. Are there any way to find the darker and lighter ones or what should we do to find right colors?

Comment (tasomaniac) [inline app/src/main/res/values/colors.xml]: So here is the API to get the colors of tags https://hub.gdgx.io/api/v1/tags/gcpnext

I am not sure if that is the right color tho. gcpnext and gcp don't have the same colors. How did you choose them @Splaktar ?

And for the darker color for the status bar, we don't have anything, you can use anything you want.

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (tasomaniac) [inline app/src/main/res/values/strings.xml]: You can get the description from hub as well.

**Comment by ouzman** [comment]
PR description updated. @tasomaniac, I've added screenshots to description of PR.

**Comment by tasomaniac** [comment]
LGTM

**Comment by tasomaniac** [comment]
If you add `Fixes #574` into your PR description, merging this will automatically close that issue

**Comment by ouzman** [comment]
I've added now.

**Comment by tasomaniac** [comment]
LGTM

**Comment by tasomaniac** [comment]
@ouzman Congrats on the first contribution to the Frisbee project.

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (Splaktar) [inline app/src/main/res/values/colors.xml]: Chosen from the respective websites colors.

**Pull request merged**